src/main/java/by/incubator/autopark/parsers/ParserVehicleFromFile.java

```diff
@@ -0,0 +1,169 @@
+    private static final String TYPES_CSV_PATH = "src/main/resources/csv/types.csv";
+    @Autowired
+    private TechnicalSpecialist technicalSpecialist;
+
+    static {
+        try {
+            initProperties();
+        } catch (IOException e) {
+            throw new RuntimeException("Error: \"csv-config.properties\" file reading has failed.", e);
+        }
+    }
+
+    public ParserVehicleFromFile() {
+
+    }
+
+    public List<VehicleType> loadTypes() throws IOException {
+        List<VehicleType> typesList = new ArrayList<>();
+        File csvFile = new File(TYPES_CSV_PATH);
+        List<String> typesBuffer = proceedStrings(readFromFile(csvFile));
+
+        for (String csvString : typesBuffer) {
+            typesList.add(createType(csvString));
+        }
+        return typesList;
+    }
+
+    public List<Rent> loadRents() throws IOException, ParseException {
+        List<Rent> rentList = new ArrayList<>();
+        File csvFile = new File(RENTS_CSV_PATH);
+        List<String> rentsBuffer = proceedStrings(readFromFile(csvFile));
+
+        for (String csvString : rentsBuffer) {
+            rentList.add(createRent(csvString));
+        }
+        return rentList;
+    }
+
+    public List<Vehicle> loadVehicles() throws IOException, ParseException {
+        List<Vehicle> vehicleList = new ArrayList<>();
+        File csvFile = new File(VEHICLE_CSV_PATH);
+        List<String> vehiclesBuffer = proceedStrings(readFromFile(csvFile));
+
+        for (String csvString : vehiclesBuffer) {
+            vehicleList.add(createVehicle(csvString));
+        }
+        return vehicleList;
+    }
+
+    private static VehicleType createType(String csvStringOfType) {
+        int id;
+        String typeName;
+        double tax;
+        String[] typeParametersBuffer = csvStringOfType.split(",");
+
+        id = Integer.parseInt(typeParametersBuffer[CONFIG.get("TYPE-CSV_ID")]);
+        typeName = typeParametersBuffer[CONFIG.get("TYPE-CSV_NAME")];
+        tax = Double.parseDouble(typeParametersBuffer[CONFIG.get("TYPE-CSV_TAX")]);
+
+        return new VehicleType(id, typeName, tax);
+    }
+
+    private static Rent createRent(String csvStringOfRent) throws ParseException {
+        int vehicleId;
+        String rentDate;
+        double cost;
+        DateFormat simpleDateFormat = new SimpleDateFormat("dd.MM.yyyy");
+        String[] rentParametersBuffer = csvStringOfRent.split(",");
+
+        vehicleId = Integer.parseInt(rentParametersBuffer[CONFIG.get("RENT-CSV_CAR_ID")]);
+        rentDate = simpleDateFormat.format(simpleDateFormat.
+                parse(rentParametersBuffer[CONFIG.get("RENT-CSV_DATE")]));
+        cost = Double.parseDouble(rentParametersBuffer[CONFIG.get("RENT-CSV_COST")]);
+
+        return new Rent(vehicleId, rentDate, cost);
+    }
+
+    private Vehicle createVehicle(String csvStringOfVehicle) throws IOException, ParseException {
+        double mass;
```

Comment:
Optional:

maybe we can somehow beatify it 
for example int id = Integer.parseInt(...);

Reply:
fixed